src/subscription_manager/repolib.py

```diff
@@ -429,7 +428,7 @@ def perform(self) -> Union["RepoActionReport", Literal[0]]:
                 if repo_file.exists():
                     log.info("Removing %s due to manage_repos configuration." % repo_file.path)
             RepoActionInvoker.delete_repo_file()
-            return 0
+            return None
```

Comment:
tbh, i'm having an hard time figuring out the return traces for this code (it's a mess)...